recipes/wx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "wxPython" %}
+{% set version = "4.0.0a2" %}
```

Comment:
Nope. The "Phoenix" version of wxPython is the first version to support Python 3, which is why many people are seeking after this package. It's still in Alpha.